Datascience_With_Python/Linear Algebra/Getting started with Linear Algebra/README.md

```diff
@@ -0,0 +1,34 @@
+<h1 align="left">Getting started with Linear Algebra</h1>
+
+## Aim:
+To give a brief introduction on what is Linear Algebra, some of its important topics ,basic examples and code snipets for simple calculation using numpy library.
+
+## What is Linear Algebra?
+<h3>A branch of mathematics dealing with linear equations.</h3>
+</br>
+
+***Some important topics of Linear Algebra in relation to Data Science:*** 
+> **1. VECTORS AND PROPERTIES** 
+
+          1.1 ADDITION
+          1.2 DOT PRODUCT
+          1.3 MAGNITUDE OF A VECTOR
+          1.4 ORTHOGONAL AND ORTHONORMAL VECTORS
+>**2. MATRICES AND PROPERTIES**
+
+          2.1 MULTIPLICATION
+          2.2 DIAGONAL MATRIX
+          2.3 IDENTITY MATRIX
+          2.4 TRANSPOSE MATRIX
+          2.5 INVERSE OF A MATRIX
+>**3. EIGEN VECTORS AND VALUES**
+
+## Applications:
+   1. Plays a major role in managing and simplifing the huge datasets required for datascience.
+   2. Linear regression can be easily found with the help of Linear algebra.
+   3. Natural Language Processing also requires Linear algebra for managing the hugwe amount of words.
+   4. Linear Algebra has a importance in Computer Vision for image processing techniques.
+
```

Comment:
Add python libraries that are being used for the demonstration of Linear Algebra.